File: 0001-two-sum/0001-two-sum.java
class Solution {
  public int[] twoSum(int[] nums, int target) {
    int length = nums.length-1;
    for (int i = 0; i <= length; i++) {
      for (int k = i+1; k <= length; k++) {
        if(nums[i] + nums[k] == target){
          return new int[]{i,k};
        }
      }
    }
    return nums;
  }
}

File: 0001-two-sum/0001-two-sum.swift
class Solution {
    func twoSum(_ nums: [Int], _ target: Int) -> [Int] {
        let l = nums.count
        
        for i in 0 ..< l {
            for j in i + 1 ..< l {
                if nums[i]+nums[j] == target{
                    return [i, j]
                }
            }
        }
        
        return []
    }
}